docs/changelog/1_0_a7.rst

```diff
@@ -0,0 +1,242 @@
+    OK: CREATE
+    db> ALTER SCALAR TYPE Color
+    ...     EXTENDING enum<Red, Green, Blue, Magic>;
+    OK: ALTER
+
+* Update :eql:type:`std::decimal` and :eql:type:`std::bigint` criteria
+  to be sensitive to the presence of ``.`` so that ``123.0e100n`` is a
+  decimal, but ``123e100n`` is a bigint (:eql:gh:`#1804`).
+
+* Functions can now return a set of tuples (:eql:gh:`#2010`).
+
+  .. code-block:: edgeql-repl
+
+    db> CREATE FUNCTION enumerate_letters(word: str)
+    ...     -> SET OF tuple<int64, str>
+    ...     USING (
+    ...         enumerate(
+    ...             array_unpack(
+    ...                 str_split(word, '')))
+    ...     );
+    OK: CREATE
+    db> SELECT enumerate_letters('hello');
+    {(0, 'h'), (1, 'e'), (2, 'l'), (3, 'l'), (4, 'o')}
+
+* Functions can no longer share a shortname with types in order to
+  avoid name resolution issues (:eql:gh:`#1465`).
+* Forbid taking implicit cross products with volatile operations to
+  avoid unintuitive behavior (:eql:gh:`#1784`).
+
+  .. code-block:: edgeql-repl
+
+    db> SELECT ({1, 2}, random());
+    error: can not take cross product of volatile operation
+      ┌─ query:1:17
+      │
+    1 │ SELECT ({1, 2}, random());
+      │                 ^^^^^^^^^ error
+    db> FOR x IN {1, 2} UNION (x, random());
+    {(1, 0.25724045818607166), (2, 0.7268530965023459)}
+
+* Forbid scalar types from having more than one concrete base
+  (:eql:gh:`#1790`).
+* Forbid partial path expressions in :ref:`LIMIT
+  <ref_eql_statements_select>`/:ref:`OFFSET
+  <ref_eql_statements_select>` clauses (:eql:gh:`#1919`).
+* Forbid changing cardinality via inheritance (:eql:gh:`#1772`).
+* Remove legacy unused ``.>`` token (:eql:gh:`#1648`).
+* Fix cardinality inference on operators (:eql:gh:`#2001`).
+
+
+Migrations
+==========
+
+We've made a lot of progress in implementing features of the `RFC 1000
+<migrations_>`_ migrations, although this is still a feature under
+development. Some of the works can be broadly categorized as overall
+improvement of the proposed migration DDL and the granularity of the
+control the user has over these proposed changes. More specifically
+we've made a lot of improvements in migrations that alter or remove
+things from the schema.
+
+Here's an example of creating a schema with a type that has a property
+with a default value:
+
+.. code-block:: edgeql-repl
+
+    db> START MIGRATION TO {
+    ...   module default {
+    ...     type Foo {
+    ...       property val -> str {
+    ...         default := 'n/a'
+    ...       }
+    ...     }
+    ...   }
+    ... };
+
+We use :eql:stmt:`DESCRIBE CURRENT MIGRATION AS JSON <DESCRIBE CURRENT
+MIGRATION>` to see what EdgeDB is proposing. The JSON format makes it
+easier to potentially integrate this with other tools. For this
+example it's worth turning on ``json`` output mode for edgedb REPL:
+
+.. code-block:: edgeql-repl
+
+    db> \set output-mode json
+    db[tx]> DESCRIBE CURRENT MIGRATION AS JSON;
```

Comment:
Why are we exhibiting raw `DESCRIBE` instead of the `create-migration` CLI?

Reply:
Largely because I'm not as familiar with create-migration CLI (since I pretty much use `DESCRIBE` in tests), but also because it is smaller in terms of explanations needed for the example, like you don't need to touch on the schema directory and file organization in it to do an example. The full-blown `create-migration` CLI example might be good fodder for the blog? Either for the release or even a standalone one?

I guess this is a work-in-progress vs. finished tool consideration. For work-in-progress showing lower level guts can be more interesting because it shows capability (like JSON that can be integrated with a tool), whereas when the tool is finished you probably want to show off how nice and smooth it is without bothering with the internals. So although it's my personal experience bias that lead me to do `DESCRIBE` in REPL example, but I think it's a good insight into our work at this stage.

Reply:
OK, fair enough.

> but I think it's a good insight into our work at this stage.

Please make this clear in the description.  `DESCRIBE CURRENT MIGRATION AS JSON` is not intended to be used directly.

Reply:
I've added another paragraph mentioning the tool and that the example is showing low-level commands.